arcade/arcade/core/schema.py

```diff
@@ -80,21 +80,24 @@ class OAuth2Requirement(BaseModel):
 class ToolAuthRequirement(BaseModel):
     """A requirement for authorization to use a tool."""
 
-    # Provider ID and Type needed for the Arcade Engine to look up the auth provider.
+    # Provider ID, Type, and ID needed for the Arcade Engine to look up the auth provider.
     # However, the developer generally does not need to set these directly.
     # Instead, they will use:
-    #    @tool(requires_auth=Google(scopes=["profile", "email"]))
+    #    @tool(requires_auth=Google(id="my_google_provider123", scopes=["profile", "email"]))
     # or
-    #    client.auth.authorize(provider=AuthProvider.google, scopes=["profile", "email"])
+    #    client.auth.authorize(provider=AuthProvider.google, id="my_google_provider123", scopes=["profile", "email"])
     #
-    # The Arcade SDK translates these into the appropriate provider ID and type.
+    # The Arcade SDK translates these into the appropriate provider id (Google) and type (OAuth2).
```

Comment:
ID can still be capitalized here, no?

```suggestion
    # The Arcade SDK translates these into the appropriate provider ID (Google) and type (OAuth2).
```